### ISSUE TITLE
Running on Python 3.5+

### ISSUE THREAD

**Issue by lenhhoxung86** [dawenl/vae_cf#6]
Hello,

I run your code with Python 3.5+ and it throws several errors. I suppose it is supposed to run with Python 2.7?
Some lines that could be edited for Python 3.5+:

uid = map(lambda x: profile2id[x], tp['userId'])
sid = map(lambda x: show2id[x], tp['movieId'])

Should be:

uid = list(map(lambda x: profile2id[x], tp['userId'])) 
sid = list(map(lambda x: show2id[x], tp['movieId']))

idxlist = range(N) should be idxlist = np.arange(N)

**Comment by dawenl** [comment]
Yes, the code is written for Python 2.7. To make it Python 3.5+ compatible, there should be mostly minor changes involving what you have mentioned here. (Maybe dict.iteritems() to the corresponding method in Python 3.)

**Comment by karthikraja95** [comment]
@lenhhoxung86 Do you have the python 3.5+ file for this repo. If so could you share that. I am struggling to make it work on python 3.5+.

Thanks

**Comment by lizhenstat** [comment]
> Yes, the code is written for Python 2.7. To make it Python 3.5+ compatible, there should be mostly minor changes involving what you have mentioned here. (Maybe dict.iteritems() to the corresponding method in Python 3.)

Hi, @dawenl , thanks for your reply. I was wondering which numpy, pandas and tensorflow version of this code? thanks a lot.
